new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -491,9 +507,15 @@ public static boolean deleteNamespace(String name) {
     }
 
     if (response.getObject() != null) {
-      logger.info(
-          "Received after-deletion status of the requested object, will be deleting namespace"
-              + " in background!");
+      V1Namespace namespace = (V1Namespace) response.getObject();
+      String phase = namespace.getStatus().getPhase();
+      logger.fine(
+          "Deletion of namespace " + namespace.getMetadata().getName() + " is in phase: " + phase);
+      if (!phase.equals("Terminating")) {
```

Comment:
I don't know how long it takes for the phase to be updated to "terminating", and how long it will stay in that phase. Wondering if this check may hit a window, and as a result, reports a false failure.

Reply:
modified code looks good.